rfc/0053-sdk3-crud.md

```diff
@@ -1201,6 +1280,18 @@ public interface ILookupInResult : IResult {
      2. Otherwise, uses the value returned by the server as the "value" in the next step. If the status code at the index is not `SUCCESS`, throws an exception matching the status code. See [RFC-0058](0058-error-handling.md) for error definitions. 
   3. Returns the value at the index, converted to the requested type.
 
+#### ILookupInReplicaResult Interface
+
+The ILookupImReplicaResult interface is the return type for Sub-Document read operations from replicas.
+
+```csharp
+public interface ILookupInResult : IResult {
+  T ContentAs<T>(int lookupIndex);
+  Boolean Exists(int lookupIndex);
+  Boolean IsReplica();
+}
```

Comment:
In Java we have GetReplicaResult extend GetResult. The analog for subdoc lookup would be for LookupInResult to extend LookupResult. The benefit of the "is-a" relationship is that code that doesn't care whether the result came from a replica can work with either a `GetResult` or a `GetReplicaResult`.

Reply:
I mostly did it this way for consistency around how IGetReplicaResult works (that also just extends? IResult).